CMakePresets.json

```diff
@@ -56,7 +56,11 @@
     },
     {
       "name": "gcc-release",
-      "configurePreset": "gcc-release"
+      "configurePreset": "gcc-release",
+      "targets": [
+        "all_verify_interface_header_sets",
+        "all"
+      ]
```

Comment:
Is there a reason why this is not applied in `gcc-debug`?

Reply:
It is verified too while building tests and examples.